%YAML:1.0



#[image]
#
#width
#1440
#
#height
#1080




Cam20250305:
  CamOldK: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d

    data: [2529.352471, 0.000000, 604.311317,
           0.000000, 2543.363241, 555.382354,
           0.000000, 0.000000, 1.000000]
  CamOldD: !!opencv-matrix
    rows: 1
    cols: 5
    dt: d
    data: [-0.505510,0.305115, -0.001399, 0.009982, 0.000000]
  
  
  CameraMat: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [2422.277100, 0.000000, 601.226789,
           0.000000, 2484.091064, 555.587087,
           0.000000, 0.000000, 1.000000]
  #topic_name:"camera/20250305"
  topic_name: "/sensing/camera/traffic_light/image_raw"
  
 
Cam20250379:
  CamOldK: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 6.5559036800000001e+02, 0., 7.6296531800000002e+02, 0., 
       6.5527504799999997e+02, 5.4024002499999995e+02, 0., 0., 1. ]


  CamOldD: !!opencv-matrix
    rows: 1
    cols: 8
    dt: d
    data: [ 4.1135149999999996e+00, 1.7315039999999999e+00,
       -4.3999999999999999e-05, -1.8900000000000001e-04,
       5.0796000000000001e-02, 4.4000029999999999e+00,
       2.9216110000000000e+00, 2.9295900000000002e-01 ]
  
  
  CameraMat: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 4.0850945999999999e+02, 0., 8.0810414500000002e+02, 0., 
       5.1616387899999995e+02, 5.4022584500000005e+02, 0., 0., 1. ]
  #topic_name: "camera/20250379"
  topic_name: "right_camera0"

Cam20250306:
  CamOldK: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [649.060688,0.000000,668.276935,
          0.000000,652.320695,505.689477,
          0.000000,0.000000,1.000000]
  CamOldD: !!opencv-matrix
    rows: 1
    cols: 8
    dt: d
    data: [0.331804, -0.041521, -0.000556, -0.000389, -0.001855 ,0.637677 ,-0.012692 ,-0.010129]
  CameraMat: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [507,0,603,
          0,507,484,
          0,0,1]
  topic_name: "left_camera0"
#    data:[391.192200 0.000000 603.823373 0.000000
#0.000000 506.419708 484.551424 0.000000
#0.000000 0.000000 1.000000 0.000000]


Cam20250376:
  CamOldK: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d

    data: [2529.352471, 0.000000, 604.311317,
           0.000000, 2543.363241, 555.382354,
           0.000000, 0.000000, 1.000000]
  CamOldD: !!opencv-matrix
    rows: 1
    cols: 5
    dt: d
    data: [-0.505510,0.305115, -0.001399, 0.009982, 0.000000]
  
  
  CameraMat: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [2422.277100, 0.000000, 601.226789,
           0.000000, 2484.091064, 555.587087,
           0.000000, 0.000000, 1.000000]
  #topic_name:"camera/20250305"
  topic_name: "/sensing/camera/hometest/image_raw"





